### DISCUSSION TITLE
Any outlines for Green Software intro talks

### DISCUSSION THREAD

**Discussion by danielvaughan** [Green-Software-Foundation/community_wg#46] [General]
Hi all,

My company has formed a Sustainable Software working group and is keen to start introducing people to Green Software principles. So far I have written an article for the Software Engineering Guild newsletter talking about the Decarbonising Software event which was a cut down version of this article: https://blog.danielvaughan.com/decarbonizing-software-51a2160ec9d2

The next step is to give a 20 minute+ talk to the Architecture Guild which will both introduce the concepts and goals of Green Software, talk about how the GSF overlaps to internal effort and encourage people to become involved in our working group or the GSF.

Rather than starting completely from scratch on a talk, which I am happy to do and share, are there any outlines or existing talks I can build on?

Thanks

Dan

**Comment by jawache** [comment]
Hey @danielvaughan, I've got this - https://docs.google.com/presentation/d/14Ii4VINo2-cCbBYzn31F0zvyyEgM5y5MlVPrEihcf5A/edit?usp=sharing  - but it's already a liiiitle dated (not updated for a year) based on an earlier version of our learn.greensoftware.org content.

**Comment by tmcclell** [comment]
Anne - some available to watch (webinars) and series of decks. @adamj89 to review potential actions for our next meeting. @NAMRATA-WOKE - list on website with up to date presentations for download and assets - media kit.
